api/registrations/serializers.py

```diff
@@ -12,13 +12,12 @@ class RegistrationSerializer(NodeSerializer):
     retracted = ser.BooleanField(source='is_retracted', read_only=True,
                                  help_text='Whether this registration has been retracted.')
     date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+    embargo_end_date = HideIfRetraction(ser.SerializerMethodField(help_text='When will embargo be lifted?'))
```

Comment:
@samchrisinger - I understand that they have different meanings in the background but since pending_registration_approval and pending_embargo give the same value, I didn't think that pending_embargo was necessary:

If I create a registration and put an embargo on it, but haven't clicked the link in the email approving the embargo, pending_embargo=True and pending_registration_approval=True.  Once I click the link in the email approving the embargo, pending_embargo=False and pending_registration_approval=False.   So I thought pending_registration_approval could handle everything.